sdk/mixedreality/mixed-reality-authentication/src/util/base64.ts

```diff
@@ -1,7 +1,8 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import { isNode } from "@azure/core-http";
+const isNode =
```

Comment:
Do we need to support browser for this library? If yes we will need to have two files: isNode.ts and isNode.brower.ts, and add a browser mapping in the package.json similar to this https://github.com/Azure/azure-sdk-for-js/blob/78f849db779d9d2045571c5dbd973f18113b2553/sdk/core/core-util/package.json#L8-L10

Reply:
@craigktreasure do you know if we support browser?

Reply:
If it supports it today (can't remember), then we should maintain support for it. If it didn't support it before, then I wouldn't bother adding it right now.

Reply:
The package already has a "browser" section in the package json file, @jeremymeng does this mean that it supports browser?

https://github.com/Azure/azure-sdk-for-js/blob/78f849db779d9d2045571c5dbd973f18113b2553/sdk/mixedreality/mixed-reality-authentication/package.json#L8-L10

Asking because there's no mentioning of broswer support on the [readme](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/mixedreality/mixed-reality-authentication/README.md) for this package, but there's [this line](https://github.com/Azure/azure-sdk-for-js/blob/78f849db779d9d2045571c5dbd973f18113b2553/sdk/mixedreality/mixed-reality-authentication/samples/v1/javascript/README.md?plain=1#L36) inside the samples readme.

Reply:
I want to say that I remember some browser specific tests.

Reply:
if the service supports/allows CORS then it is good!  We run browser tests with security disabled.